cql3/statements/describe_statement.cc

```diff
@@ -0,0 +1,856 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ *
+ * Modified by ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: (AGPL-3.0-or-later and Apache-2.0)
```

Comment:
Code that isn't derived from Cassandra shouldn't be licensed under Apache 2.0. The "modified by ScyllaDB" isn't needed in that case.

Reply:
fixed